tests/python/relay/test_dataflow_pattern.py

```diff
@@ -1033,6 +1033,18 @@ def conv_bias_relu(x, w, b):
     assert pattern2.match(relu)
     assert tvm.ir.structural_equal(func(x, w, b), pattern2.partition(relu))
 
+def test_match_match():
+    add_pattern = is_op('add')(wildcard(), wildcard())
+    class TestRewrite(DFPatternCallback):
+        def __init__(self):
+            self.pattern = add_pattern
+        def callback(self, pre, post, node_map):
+            return post.args[0] - post.args[1]
+    mod = tvm.IRModule({})
+    tvm.relay.prelude.Prelude(mod)
+    # Apply rewrite on IR including relay.Match
+    rewrite(TestRewrite(), mod['tensor_concatenate_int64'])
```

Comment:
Can you add an assert on the output of this? I understand the goal is to write something that will catch a rewrite overflow, but I'd prefer to check the output as well.

Reply:
I've fixed this